.github/mergify.yml

```diff
@@ -0,0 +1,42 @@
+pull_request_rules:
+- actions:
+    merge:
+      strict: smart+fasttrack
+      method: squash
+  name: Automatically merge pull requests
+  conditions:
+  - status-success=bench (8.10.2, ubuntu-latest)
+  - status-success=bench (8.6.5, ubuntu-latest)
+  - status-success=bench (8.8.4, ubuntu-latest)
+  - status-success=nix (default, macOS-latest)
+  - status-success=nix (default, ubuntu-latest)
+  - status-success=test (8.10.3, ubuntu-latest)
+  - status-success=test (8.10.3, windows-latest)
+  - status-success=test (8.10.2, ubuntu-latest)
+  - status-success=test (8.10.1, ubuntu-latest)
+  - status-success=test (8.10.1, windows-latest)
+  - status-success=test (8.8.4, ubuntu-latest)
+  - status-success=test (8.8.3, ubuntu-latest)
+  - status-success=test (8.8.2, ubuntu-latest)
+  - status-success=test (8.6.5, ubuntu-latest)
+  - status-success=test (8.6.5, windows-latest)
+  - status-success=test (8.6.4, ubuntu-latest)
+  - status-success=test (windows-latest, 8.10.2.2)
+
+  - 'status-success=ci/circleci: ghc-8.10.1'
+  - 'status-success=ci/circleci: ghc-8.10.3'
+  - 'status-success=ci/circleci: ghc-8.6.4'
+  - 'status-success=ci/circleci: ghc-8.8.2'
+  - 'status-success=ci/circleci: ghc-8.8.3'
+  - 'status-success=ci/circleci: ghc-8.10.2'
+  - 'status-success=ci/circleci: ghc-8.6.5'
+  - 'status-success=ci/circleci: ghc-8.8.4'
+  - 'status-success=ci/circleci: ghc-default'
+
+  - label=merge me
+  - '#approved-reviews-by>=1'
+- actions:
+    delete_head_branch: {}
+  name: Delete head branch after merge
+  conditions:
+  - merged
```

Comment:
Sorry for late reply, but if we use squash for merge and delete the head branch of PR, we will lose commit log, so I think we should not delete headbranch (at least automatically) if we use `squash`.

Reply:
right, thanks for noting it, contributors still could delete them though

Reply:
Yeah, manual deletion will make more sense then automatic one under this setting, I think.